### PR TITLE
Voxel sample rate

### PR DESCRIPTION
**Pull request by alexDrinkwater**

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -7427,6 +7434,13 @@
             "Generated",
             "UserDefined"
           ]
+        },
+        "voxelSampleRate": {
+          "description": "the amount of sub voxels per voxel in each direction (x,y,z)",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 1,
+          "maximum": 10
```
Comment (pkrull) [inline swagger.json]: I'm not sure how this file got included in this repo. I'm not a fan of having it in here. While it (theoretically) provides the swagger the client was created from, it gives the wrong impression that this is the canonical specification when in fact simulation-api-specification contains the real spec. I'd prefer we not include it in this repo.

**Review by pkrull — CHANGES_REQUESTED** [review]

File: swagger.json (hunk 1 above)
Comment (pkrull) [reply swagger.json]: I'm open to negotiation on this.

Comment (alexDrinkwater) [reply swagger.json]: Understand, you were the one that added it originally so though I would follow.  I will remove it.

Comment (pkrull) [reply swagger.json]: It figures...

**Review by pkrull — APPROVED** [review]

**Pull request merged**